test/test_other.py

```diff
@@ -7498,6 +7501,21 @@ def test(page_diff):
     # default maximum memory is 2GB.
     self.assertEqual(less, none)
 
+  @parameterized({
+    '': (['-DWORKERS=1'],),
+    'pthreads': (['-pthread', '-DWORKERS=4'],),
+  })
+  def test_mimalloc(self, args):
+    args = args + [
+      '-O2',
+      '-sMALLOC=mimalloc',
+      '-sINITIAL_MEMORY=128mb',
+      '-sTOTAL_STACK=1mb',
+      '-sEXIT_RUNTIME',
+      '-DTOTAL=10000',
+    ]
+    self.do_other_test('test_mimalloc.cpp', emcc_args=args)
```

Comment:
At I see this is both a benchmark and an "other" test.   This still seems like a test that could be run with all mallocs, not just mimalloc.

Reply:
Correct, yes. But see my comment above, it is focused on mimalloc because it is a multithreaded malloc test which only mimalloc really does properly.